src/test/ui/suggestions/enum-method-probe.rs

```diff
@@ -0,0 +1,42 @@
+// run-rustfix
```

Comment:
Let's add a check for `async fn` as well. I'm intrigued if they work without further changes.